source/CommonJobs/CommonJobs.Application.MyMenu/GetEmployeeMenuCommand.cs

```diff
@@ -47,13 +47,22 @@ public override EmployeeMenuDTO ExecuteWithResult()
 
             var menuDefinition = ExecuteCommand(new GetMenuDefinitionCommand(employeeMenu.MenuId));
 
-            var lastOrder = RavenSession.Load<MenuOrder>(MenuOrder.GenerateId(employeeMenu.MenuId, menuDefinition.LastOrderDate));
-            if (lastOrder != null)
-            {
-                lastOrder.IsOrdered = true;
-            }
+            var today = DateTime.Now.Date;
```

Comment:
I think that in general we was trying to make commands non-dependent on `DateTime.Now`, what do you think about to send current date as a parameter?